src/util/unit.ts

```diff
@@ -0,0 +1,8 @@
+/**
+ * Enum representing units, e.g. Kilogram, the base unit of mass.
+ * @readonly
+ * @enum
+ */
+export enum Unit {
+    Kilogram = 'kg',
```

Comment:
if you want to support different units in the future, this would need to support unit conversion

Reply:
Since it's kg in the backend, I will keep it simple and only support kg.